README.md

```diff
@@ -0,0 +1,48 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+Creates a Lambda function with associated role and policies that
+will run the packer-janitor tool to get rid of orphaned Packer
```

Comment:
This is picky but you use packer-janitor in some places and packerjanitor in others. I think it has to do with when you're using underscores so you're probably trying to avoid comboing underscores with hyphens. I guess unless you have a strong rationale, I'd say stick with one.

Reply:
So, the difference here is "packer-janitor" is the Go tool that the lambda job "packerjanitor" runs...does that make more sense?

Reply:
I changed the instance where I was calling the module "packer-janitor-lambda" since you're right, that seemed inconsistent with usage elsewhere.